test/docker-compose-with-error.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+usage() { echo "Usage: $0 -f docker-compose.yml" 1>&2; exit 1; }
```

Comment:
`SERVICES` isn't documented here.

Also, it looks like this only accepts a single `-f` argument, correct? If so, that might be good to call out.